sections/ads.txt

```diff
@@ -779,6 +779,7 @@ mno.hu##[id*="Banner"]
 mno.hu##[id*="ctnet"]
 mobilarena.hu##.adbox
 mobilarena.hu##[class*="adtext"]
+mobilarena.hu###mobilarena_roadblock_1:upward(#center > div)
```

Comment:
Just a quick note, `:upward` pseudo-class isn't compatible with Adblock Plus. In the root of the `sections` folder, we collect "cross-compatible rules" that work with Adblock Plus, AdGuard, and uBlock. Additionally, we maintain separate subfolders within the `sections` folder for "adblocker-specific" rules.